src/styles/product-card.scss

```diff
@@ -0,0 +1,94 @@
+.product {
+  width: 200px;
```

Comment:
Consider applying paddings here to make 100% width on the button work properly. It's bad practice to hardcode values, that should depend on the content or the parent container width

Reply:
I did it in such way: `width: calc(100% - 32px);`

Reply:
This was in the requirements: 

> card width is 200px including border